app/components/candidate_interface/referees_review_component.html.erb

```diff
@@ -9,9 +9,17 @@
 <% @application_form.application_references.includes(:application_form).each do |referee| %>
   <%= render(SummaryCardComponent.new(rows: referee_rows(referee), editable: @editable)) do %>
     <%= render(SummaryCardHeaderComponent.new(title: t('application_form.referees.nth_referee')[referee.ordinal], heading_level: @heading_level)) do %>
-      <% if @editable %>
+
+      <% if FeatureFlag.active?('candidate_can_cancel_reference') && referee.feedback_status != 'cancelled' %>
```

Comment:
This needs some thought - I think we only want to allow candidates to cancel references in very specific circumstances. This will show the cancel link when the application isn't submitted yet, and also when the feedback already has been provided and the application has been sent to the provider.

I *think* the logic should be that candidates can only cancel a reference that's in the `feedback_requested` state?

We also need to make sure we check the state in the controller before showing the confirmation / cancelling the reference for real.

Reply:
Good point. I've redirected them to the referee review page if it's not in the correct state. If they've submitted the before action will redirect them to the complete page.